community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/variables.tf

```diff
@@ -264,6 +264,24 @@ variable "disable_default_mounts" {
   default     = false
 }
 
+variable "enable_controller_default_mounts" {
+  description = <<-EOD
+    Enable default global network storage to controller.
+    - /home
+    - /apps
+  EOD
+  type        = bool
+  default     = true
+}
+
+variable "controller_home" {
+  description = <<-EOD
+    GCS bucket path to mount controller home directory across the cluster.
```

Comment:
is it the controller home though? or is it a home directory that is used for all the nodes in the cluster? this description is not very clear.